src/Directory.Build.targets

```diff
@@ -1,4 +1,9 @@
 <Project>
+  <PropertyGroup Condition="'$(MinVerTagPrefix)' == 'core-' AND '$(RunPackageValidation)' == 'true' AND '$(RunApiCompat)' == 'true'">
```

Comment:
> Rather than introducing a new property RunPackageValidation, consider just using the existing one: EnablePackageValidation

Are you suggesting to set the value of `EnablePackageValidation` in the workflow as it is now, and then here do like this?

```xml
<PropertyGroup Condition="'$(MinVerTagPrefix)' == 'core-' AND '$(EnablePackageValidation)' == 'true' AND '$(RunApiCompat)' == 'true'">
    <PackageValidationBaselineVersion>$(OTelLatestStableVer)</PackageValidationBaselineVersion>
</PropertyGroup>
```

Reply:
I see some value in introducing the property `RunPackageValidation`. This would let us skip package validation check when actually [publishing packages](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/.github/workflows/publish-packages-1.0.yml). If we run package validation check for every PR then we can be assured that the workflow for actually publishing the package doesn't have to run it again.

Reply:
When doing your publishing package step, you could just pass in `/p:EnablePackageEvalidation=false` to make sure it doesn't run.

Reply:
Yes that's also an option. Another nice perk about having the additional property is that the package validation configuration stuff remains all in one place.

@utpilla what do you think? Shall we leave it as is or remove the introduced property and do like Viktor is suggesting?

Reply:
@mfogliatto I think we could just use the `EnablePackageValidation` property instead of introducing one ourselves.

@ViktorHofer Does the .NET Package Validation tool print out any output? It would be great if we could look at the build output in the CI workflow to check if the tool actually ran. Currently, it doesn't seem to log anything related to validation at all.

Reply:
It doesn't as msbuild by default runs in `minimal` verbosity mode. If you want it to print a confirmation message, you would need to change the msbuild verbosity level globally.

Note that many other tools run as part of the build, not just Roslyn and NuGet. You don't see confirmation messages for their run either. The default for modern .NET is to not emit a high priority message unless necessary.

If you set the `EnablePackageValidation` property to true, PackageValidation/APICompat will always run after the Pack target.

Reply:
Agreed to use the existing property instead of adding a new one. Resolving the comment.